composer.json

```diff
@@ -32,39 +32,39 @@
         "doctrine/doctrine-fixtures-bundle": "^2.3",
         "doctrine/doctrine-migrations-bundle": "^1.2",
         "doctrine/orm": "^2.5",
-        "friendsofsymfony/oauth-server-bundle": "^1.5",
+        "friendsofsymfony/oauth-server-bundle": "^1.6@dev",
         "friendsofsymfony/rest-bundle": "^2.1",
         "fzaninotto/faker": "^1.6",
         "gedmo/doctrine-extensions": "^2.4.12",
-        "incenteev/composer-parameter-handler": "^2.1",
+        "incenteev/composer-parameter-handler": "^2.1.3",
         "jms/serializer-bundle": "^2.0",
         "knplabs/knp-gaufrette-bundle": "^0.5",
         "knplabs/knp-menu-bundle": "^2.1",
-        "liip/imagine-bundle": "^1.9.1",
+        "liip/imagine-bundle": "^2.0@dev",
         "ocramius/proxy-manager": "^2.1",
         "payum/payum": "^1.4",
         "payum/payum-bundle": "^2.2",
         "php-http/guzzle6-adapter": "^1.1",
         "polishsymfonycommunity/symfony-mocker-container": "^1.0",
         "ramsey/uuid": "^3.7",
-        "sensio/distribution-bundle": "^5.0.21",
         "sonata-project/block-bundle": "^3.3",
         "sonata-project/intl-bundle": "^2.2",
         "stof/doctrine-extensions-bundle": "^1.2",
         "swiftmailer/swiftmailer": "^6.0",
-        "sylius-labs/association-hydrator": "^1.0",
+        "sylius-labs/association-hydrator": "^1.1",
+        "sylius-labs/sensio-distribution-bundle": "^6.0",
         "symfony/monolog-bundle": "^3.0",
         "symfony/polyfill-iconv": "^1.3",
         "symfony/polyfill-intl-icu": "^1.3",
         "symfony/polyfill-mbstring": "^1.3",
         "symfony/swiftmailer-bundle": "^3.0",
-        "symfony/symfony": "^3.4",
+        "symfony/symfony": "^4.0",
         "symfony/thanks": "^1.0",
         "twig/extensions": "^1.4",
         "twig/twig": "^2.0",
         "webmozart/assert": "^1.1",
         "white-october/pagerfanta-bundle": "^1.0.8",
-        "willdurand/hateoas": "~2.10.0",
+        "willdurand/hateoas": "^2.11@dev",
```

Comment:
2.11.0 release introduces a regression from 2.10.0 and it should be fixed and merged to 1.0 first.